understanding/21/non-text-contrast.html

```diff
@@ -83,8 +83,82 @@ <h4 id="related-scs">Relationship with Use of Color and Focus Visible</h4>
+					<figcaption>
+						The focus indicator is partially inside, partially outside the button. The internal part of the yellow indicator (#FFFF00) contrasts with the blue button background (#4189B9), <strong>passing</strong> the criterion.
+					</figcaption>
+				</figure>
+
+				<p>If the focus indicator changes the border of the component within the visible boundary it must contrast with the component. Typically an outline goes around (outside) the visible boundary of the component, in this case changing the border is just inside the visible edge of the component.</p>
+
+				<figure id="figure-focus-border">
+					<img src="img/ntc-focus-border.png" alt="Three blue buttons on a white background, the center button has a green border exactly on the outer boundary of the button." width="400" />
+					<figcaption>
+						The border of the control changes from blue (#4189B9) to green (#4B933A). This is within the component and does not contrast with the inside background of the component therefore <strong>fails</strong> non-text contrast.
+					</figcaption>
+				</figure>
+
+				<figure id="figure-focus-inner-green">
+					<img src="img/ntc-focus-inner-border.png" alt="Three blue buttons with a black border on a white background, the center button has a green border inside, adjacent to the inner background and black border." width="400" />
+					<figcaption>
+						An inner border of dark green (#008000) does contrast with the black border, but does not contrast with the blue component background, therefore <strong>fails</strong> non-text contrast.
+					</figcaption>
+				</figure>
+
+				<figure id="figure-focus-inner-green">
+					<img src="img/ntc-focus-inner-white.png" alt="Three blue buttons with a black border on a white background, the center button has a white border inside, adjacent to the inner background and black border." width="400" />
+					<figcaption>
+						An inner border of white contrasts with the black border and the blue component background, therefore <strong>passes</strong> non-text contrast.
+					</figcaption>
+				</figure>
+
+		        <p>Note that this Success Criterion does not directly compare the focused and unfocused states of a control - if the focus state relies on a change of color (e.g., changing <em>only</em> the background color of a button), this Success Criterion does not define any requirement for the difference in contrast between the two states.</p>
+
+				<figure id="figure-focus-background">
+					<img src="img/ntc-focus-background.png" alt="Three blue buttons, the center button is a lighter blue than the others." width="400" />
+					<figcaption>
+						The change of background within the component is not in scope of non-text contrast. However, this would not pass <a href="../20/use-of-color.html">Use of color</a>.
```

Comment:
didn't notice this at the time, but revisiting the google doc: while the change of just the background would fail use of color, the text of the button changes from white to black - the most dramatic change in contrast, which would not count as a change of color alone...so this would *not* fail Use of color?

Reply:
made this into a fresh issue https://github.com/w3c/wcag/issues/2061